src/helpers.js

```diff
@@ -37,6 +41,11 @@ export function toReadableNumber(number) {
     }
 }
 
+export function localize_datetime(time) {
```

Comment:
Not very important but we could also consume timeformat from src/lib.

Reply:
Eventually, yes. But timeformat does not yet wrap `formatRelative()`, so we need to add that first. Indeed I replaced a few moment.calendar() with absolute times, I think it'd be nice to bring these back.